### ISSUE TITLE
time_series_covid19_recovery_global.csv is not part of the directory 

### ISSUE THREAD

**Issue by therealcyberlord** [CSSEGISandData/COVID-19#1498]
Can you guys please add the number of recoveries to time series data? Thanks.

**Comment by downFast** [comment]
Me too I am looking for Recovery dataset confirmed global

**Comment by bats66** [comment]
Yes I second this request

**Comment by Niaashaa** [comment]
If it is not provided,  I will use estimation (21day lag for confirmed case - death)

**Comment by downFast** [comment]
They say they will not provide any longer if not on the daily one as per https://github.com/CSSEGISandData/COVID-19/issues/1250

Yet why?

**Comment by clyde7** [comment]
They answered in Issue #1250: "No reliable data source reporting recovered cases for many countries, such as the US."

**Comment by downFast** [comment]
> They answered in Issue #1250: "No reliable data source reporting recovered cases for many countries, such as the US."

Ok but why do they still show it then? https://www.arcgis.com/apps/opsdashboard/index.html#/bda7594740fd40299423467b48e9ecf6

Shall we rea all the daily report, get the recovered data and do a sum for us to display? That's crazy

**Comment by Niaashaa** [comment]
They have it on daily file.  Need proper ETL to change it to this format.
https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_daily_reports

**Comment by downFast** [comment]
> They have it on daily file. Need proper ETL to change it to this format.
> https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_daily_reports

yes I saw it but I'd need it as a single file with all dates as per Deaths and Confirmed, can anyone make it and put it on git please?

**Comment by JNFoo** [comment]
For US by State (not global, unfortunately), another poster mentioned using https://covidtracking.com/api/states/daily.csv  
The data is very similar. There are some timing differences vs JHU, but it is created by a team that helped The Atlantic.

**Comment by chschoenenberger** [comment]
> 
> 
> If it is not provided, I will use estimation (21day lag for confirmed case - death)

That's a great idea! I'll probabily use coronadatascraper.com/ in the future

**Comment by geoced** [comment]
They still provide recoveries numbers on their dashboard and in their daily report CSV files, so I really don't see any reason not to provide it in a global time series...

**Comment by downFast** [comment]
Me too. I don't understand it, you have why not show it?
